env0/resource_environment.go

```diff
@@ -1113,6 +1115,21 @@ func resourceEnvironmentImport(ctx context.Context, d *schema.ResourceData, meta
 	d.Set("deployment_id", environment.LatestDeploymentLogId)
 	setEnvironmentSchema(d, environment, environmentConfigurationVariables)
 
+	if environment.IsRemoteBackend != nil {
```

Comment:
the following have tfschema disabled (for various/historical reasons).
setting them manually.